contrib/localnet/preparams/zetaclient-new-validator.json

```diff
@@ -0,0 +1 @@
1958433921811834519383439615652125566062108517595254434202585520725918458539650829611359906765649588876933069244900503743779957722205401652112799388769959315574963861799346623992179296812259108512120682902363605777290798387279703060866846190527558754540944004025669889182402130711051994547424947465848311097872058264068264847883302814930546814509196138468589465110378896941398104531133441169885749142287817235614162212537129367684226889834353016992347371767689577099973152384856154921470842340609440,"H2i":13248076109800665469593810622719225502034768318277483018263693642187250618832552829080364379294987051479503793767958094397864222449722505944252701999223425055634923926237370470982156904748484641055882829739274419766142360133464208567250462282877978030879964096066657152108997164557232745990125568297265347801334911454843872417470925322427408725733589015190657409390596754236317010520232686262328847585710830812639495417877966446490298931635599170573064212526961050707936186722270771009657726606965398426725913421261248838826460745873909371448250697598806763783080833425863062403039913803567383221386972419419382976826,"Alpha":18462405526887919865545155849471766186599587820278228502134585515184526131150630703061956418002947703995062977027748808417033945513306979470746255795774997923957337702604617215611885646144358621346101126449123083330067178276213967454833418565658278757978219923186764710442221649754942924977610640847987230019839733787728557189291779267774518176250222724619645559678442357085927681807757055805549792941365193910082816045554295507286321570914925968892626033251553365817524206817760323640196878009227545160591446053818315974810547625672619543716071058287058884144816630763414174708650897710662822999640645173800018653221,"Beta":1429913922854495484075164077172231537041798673392692054168381388920937923051472026736600648681839975302763780951378731902324765131763911908568833503423773456149523938164072252282486164059731813002194341978761252717289831153831286393667796900316723580171967625417278152306356668052770617502747079397572606559275307222452666125825842415793107024193944337960680527312794164583189658388348765076298799696009572118455188779414852680237339024760719259079238031237542103736413560790515107480231245162701093410860792468523251918769151182358823233381431993231319450443741382536143710728710355878800739118210773121451375904013,"P":72198038811472636386819145284002882962985025633289600640684386221873161724939238359793709956852317052085086520668678160660824485742685496099068043857977938326882557402973989895480775128070314428066843983321552127922793878258934593560477986131553237785383592836252607797792226456757856580055145941688232468861,"Q":84007205688389688038268246876633594660472993846606243250776906606448406376486738741891900213586069402455906606766420548280187082244432211112119353536676511843676678178417693927680436975944782140020243974926366756213564009523117668835831522979261286060994164078613742602246511159941849840256385604289986923081}
```

Comment:
Where does it comes from?

Reply:
It's a copy of the pre-params used by validator 3